rules/docker-lldp-sv2.mk

```diff
@@ -2,7 +2,7 @@
 
 DOCKER_LLDP_SV2 = docker-lldp-sv2.gz
 $(DOCKER_LLDP_SV2)_PATH = $(DOCKERS_PATH)/docker-lldp-sv2
-$(DOCKER_LLDP_SV2)_DEPENDS += $(LLDPD)
+$(DOCKER_LLDP_SV2)_DEPENDS += $(LLDPD) $(LIBSWSSCOMMON) $(PYTHON_SWSSCOMMON)
```

Comment:
Do you need LIBSWSSCOMMON here?

Reply:
it uses python-binding of libswsscommon for getting table notification from redis db.

Reply:
ok

Reply:
ok